src/main/java/com/synopsys/integration/blackduck/api/generated/discovery/ApiDiscovery.java

```diff
@@ -37,7 +37,7 @@
 import com.synopsys.integration.blackduck.api.generated.view.OriginDependencyPathsView;
 import com.synopsys.integration.blackduck.api.generated.view.PolicyRuleView;
 import com.synopsys.integration.blackduck.api.generated.view.ProjectGroupsView;
-import com.synopsys.integration.blackduck.api.generated.view.ProjectView;
+import com.synopsys.integration.blackduck.api.manual.view.ProjectView;
```

Comment:
The ProjectView class is moved to Manual package from Generated while HUB specification remains inconsistent.